workflow/lambdas/mvl/Dockerfile

```
@@ -0,0 +1,25 @@
+# Use AWS Lambda Node.js 22 base image
+FROM public.ecr.aws/lambda/nodejs:22
+
+# Install git (required for npm to install from GitHub)
+# Combine RUN commands and clean up in same layer to reduce image size
+RUN dnf install -y git && dnf clean all && rm -rf /var/cache/dnf
```

Hadolint (2.14.0)

[warning] Specify version with `dnf install -y <package>-<version>`.

(DL3041)